azure-pipelines.yml

```diff
@@ -313,17 +313,12 @@ jobs:
       targetType: 'inline'
       script: 'Install-Module -Name Az.Storage -RequiredVersion 1.11.0 -Scope CurrentUser -Force -AllowClobber'
 
-  - task: AzureKeyVault@1
```

Comment:
With this change, pulling secrets from this KV is no longer required. We can delete this after merging.